pkg/util/configmap.go

```diff
@@ -0,0 +1,56 @@
+// Package util provides shared utility functions used across the ToolHive codebase.
```

Comment:
Put this in a `k8s` package or something similar. `util` packages tend to turn into a dumping ground for random functionality.